storage/aptosdb/src/lib.rs

```diff
@@ -259,13 +261,15 @@ impl AptosDB {
     fn new_with_dbs(
         ledger_rocksdb: DB,
         state_merkle_rocksdb: DB,
+        kv_rocksdb: DB,
         pruner_config: PrunerConfig,
         buffered_state_target_items: usize,
         max_nodes_per_lru_cache_shard: usize,
         hack_for_tests: bool,
     ) -> Self {
         let arc_ledger_rocksdb = Arc::new(ledger_rocksdb);
         let arc_state_merkle_rocksdb = Arc::new(state_merkle_rocksdb);
+        let arc_kv_rocksdb = Arc::new(kv_rocksdb);
```

Comment:
Let's make it optional and configurable? (on a node in production don't create the DB folder and allocate db threads)

Reply:
Done